action.yml

```diff
@@ -6,18 +6,27 @@ inputs:
     description: 'Which version of the Dune binary to install'
     required: false
     default: 'nightly'
-  automagic:
-    description: 'Whether to run automatically all the standard lock, build, test steps'
-    required: false
-    default: 'false'
   steps:
     description: 'Fine-grain control of the steps to run; override `automagic` when set'
```

Comment:
This needs to be updated, because automagic does not exist anymore. I do think this is a clearer way than having both steps and `automagic`!